src/main/java/com/parsing/client/ProzorroClient.java

```diff
@@ -0,0 +1,52 @@
+package com.parsing.client;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RequestCallback;
+import org.springframework.web.client.ResponseExtractor;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.File;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class ProzorroClient {
+
+    private final RestTemplate restTemplate;
+    private static final String DIR_TO_SAVE_PDF = "/resources/";
```

Comment:
should it be a dedicated folder in resources or it's temporary and no mess is expected?

Reply:
changed to PDF.